kong/plugins/pre-function/_handler.lua

```diff
@@ -42,6 +43,38 @@ return function(plugin_name, priority)
     end
   end
 
+  function ServerlessFunction:new()
+    ServerlessFunction.super.new(self, "ServerlessFunction:" .. plugin_name)
+  end
+
+  function ServerlessFunction:init_worker(config)
```

Comment:
there is no config parameter here. Init_worker runs on a "process" event, not a "request" event.